### DISCUSSION TITLE
My repo got disabled for no reason and I submitted a appeal and opened a ticket but it has been 2 weeks and not a single reply

### DISCUSSION THREAD

**Discussion by zSynctic** [community/community#144743] [Repositories]
### Select Topic Area

General

### Body

My repo got disabled for no reason and I submitted a appeal and opened a ticket but it has been 2 weeks and not a single reply when I never did anything wrong also the repo has been there for a year or 2 but suddenly got removed

**Comment by davevad93** [comment]
Hi @zSynctic ,

Generally a repo is removed from GitHub when the [Terms of Service](https://docs.github.com/en/site-policy/github-terms/github-terms-of-service) has been violated. To know exactly what happened you have to wait for a response from GitHub Support staff, they can take a while to reply, even weeks because they have a lot of daily requests to deal with, so be patient. I hope you get a resolution as soon as possible.

**Reply by zSynctic** [reply]
It has been 2 months its literally a shame my program had around 40k downloads and suddenly got removed for no reason
